Cargo.toml

```diff
@@ -18,9 +18,10 @@ serde = "~1.0"
 serde_derive = "~1.0"
 serde_yaml = "~0.8"
 serde_json = "~1.0"
-futures = "~0.1"
-hyper = "~0.12"
+futures = "~0.3"
+hyper = {version= "~0.14", features = ["full"]}
```

Comment:
Does this really need `full` features? Same with tokio. I'm wondering if the feature list can be parred down a little.

Reply:
Technically I think I need everything except the 'server' feature, but let me get back to you on that